pylops/utils/decorators.py

```diff
@@ -121,3 +121,42 @@ def wrapper(self, x):
     if func is not None:
         return decorator(func)
     return decorator
+
+
+def count(f=None, forward=True, mat=False):
```

Comment:
Was it a design choice to not derive forward/mat based on the `f.__name__` as was done in the `reshaped` decorator? If it wasn't and you think it's a good idea, let me know and I can implement it.

Reply:
Mmh not really. I think it makes sense to try to be as consistent as possible between different routines in the library, so I fixed it. Have a look and let me know if you agree.

Question: shall I still be verbose in `LinearOperator` or not (I mean shall I just use @count or specify the parameters? What do you suggest? For now I don't change it, but I tried and the new code would work as well by doing the internal evaluation based on `f.__name__`

Reply:
Awesome, the change looks good. Regarding your question, personally I would remove all the arguments, but that's my personal preference.